api/src/opentrons/protocol_api/instrument_context.py

```diff
@@ -1447,22 +1461,32 @@ def tip_racks(self, racks: List[labware.Labware]) -> None:
 
     @property  # type: ignore
     @requires_version(2, 0)
-    def trash_container(self) -> labware.Labware:
-        """The trash container associated with this pipette.
+    def trash_container(
+        self
+    ) -> Union[labware.Labware, TrashBin, WasteChute]:
+        """The trash container(s) associated with this pipette.
```

Comment:
```suggestion
        """The trash containers associated with this pipette.
```
Always favor the plain plural over `(s)`.

If we want to be even more explicit about this object, and further moot the plural issue: `The list of any trash containers...`